lib/mrsk/commands/builder.rb

```diff
@@ -1,3 +1,5 @@
+require "active_support/core_ext/string/inquiry"
```

Comment:
This is already included in `lib/mrsk/configuration.rb`. Looks like we need to include `active_support/core_ext/string/filters` there as well now